src/andamio/course/101/1013.md

```diff
@@ -38,37 +40,43 @@ export default function MyWalletAssetComponent() {
 ```
 
 ### Step 3: Add the Component to Your Application
+
 In this example, we will continue to build on the main page of the app. Open `/src/pages/index.tsx` and find this part:
+
 ```typescript
 <div className="demo">
   <MyWalletComponent />
 </div>
 ```
 
 Below these lines, add the following:
+
 ```typescript
 <div className="demo">
   <MyWalletAssetComponent />
 </div>
 ```
 
 At the top of the file, find this line:
+
 ```typescript
 import MyWalletComponent from "../components/MyWalletComponent";
 ```
 
 And add a new import statement below it:
+
 ```typescript
 import MyWalletAssetComponent from "../components/MyWalletAssetComponent";
 ```
 
-Use `npm run dev` to start your development server. In a web browser, open [localhost:3000](http://localhost:3000). You should now see two copies of the same component on the page:
+Use `npm run dev` to start your development server. In a web browser, open `http://localhost:3000`. You should now see two copies of the same component on the page:
 
 ![](/course/images/101/two-components.png)
```

Comment:
Same question to 100's review, do we miss something here or we would like it to a production link later?

Reply:
Localhost is good here - we're asking students to try the student repo

Reply:
I mean the line 74, a bit confusing the comment sorry.

> `![](/course/images/101/two-components.png)`